rosbag2_storage_mcap/src/mcap_storage.cpp

```diff
@@ -534,26 +534,63 @@ void MCAPStorage::reset_iterator()
     options.endTime = mcap::MaxTime;
   }
   options.readOrder = read_order_;
-  if (!storage_filter_.topics.empty()) {
-    options.topicFilter = [this](std::string_view topic) {
+
+  auto filter_process = [this](std::string_view topic) {
```

Comment:
Comments from MichaelOrlov

```suggestion
  auto topic_filter = [this](std::string_view topic) {
```
IMO the filter_process name is misleading since it is not running in a separate process.

Reply:
Updated

Reply:
@Barry-Xu-2018 Please rename `filter_topic` to the `topic_filter`. There are subtle differences in the word order in name.